apitools/org.eclipse.pde.api.tools/src/org/eclipse/pde/api/tools/internal/search/UseReportConverter.java

```diff
@@ -1186,11 +1181,10 @@ protected void writeReferencedMemberPage(final Report report, final List<Type> r
 			buffer.append("\t<td bgcolor=\"").append(REFERENCES_TABLE_HEADER_COLOUR).append("\" width=\"10%\" align=\"center\">").append(OPEN_B).append(SearchMessages.UseReportConverter_reference_count).append(CLOSE_B).append(CLOSE_TD); //$NON-NLS-1$ //$NON-NLS-2$
 			buffer.append(CLOSE_TR);
 			Collections.sort(referees, compare);
-			IComponentDescriptor comp = null;
-			for (int i = 0; i < referees.size(); i++) {
-				type = referees.get(i);
-				comp = (IComponentDescriptor) type.desc;
-				buffer.append("<tr bgcolor=\"").append(getRowColour(counts)).append("\">\n"); //$NON-NLS-1$//$NON-NLS-2$
+			for (Type referee : referees) {
+				Type type = referee;
+				IComponentDescriptor comp = (IComponentDescriptor) type.desc;
+				buffer.append("<tr bgcolor=\"").append(getRowColour(type.counts)).append("\">\n"); //$NON-NLS-1$//$NON-NLS-2$
```

Comment:
This used to be the last counts from the loop above. I think this was a bug.

In general I wonder who thought that it would be good style to declare all variables in the beginning...